openpaas-james/apps/memory/src/main/java/com/linagora/openpaas/james/app/MemoryServer.java

```diff
@@ -49,19 +59,37 @@
         new SpamAssassinListenerModule());
 
     public static void main(String[] args) throws Exception {
-        Configuration configuration = Configuration.builder()
+        MemoryConfiguration configuration = MemoryConfiguration.builder()
             .useWorkingDirectoryEnvProperty()
             .build();
 
         LOGGER.info("Loading configuration {}", configuration.toString());
-        GuiceJamesServer server = createServer(configuration)
+        GuiceJamesServer server = GuiceJamesServer.forConfiguration(configuration)
+            .combineWith(MODULES)
+            .overrideWith(chooseMailbox(configuration.mailboxConfiguration()))
             .combineWith(new FakeSearchMailboxModule(), new JMXServerModule());
 
         JamesServerMain.main(server);
     }
 
-    public static GuiceJamesServer createServer(Configuration configuration) {
+    public static GuiceJamesServer createServer(MemoryConfiguration configuration) {
```

Comment:
This become not used and useless?

Reply:
Used by test classes so should be the other way around to avoid the code duplication